From c99d6b760a55d4ad7837141cb8c457e93914273e Mon Sep 17 00:00:00 2001
From: Boris Brezillon <boris.brezillon@collabora.com>
Date: Wed, 15 Oct 2025 10:20:29 +0200
Subject: [PATCH 108/161] drm/tyr: Fix version

1.0 was the first version to be exposed, and mesa assumes at least that.
---
 drivers/gpu/drm/tyr/driver.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/tyr/driver.rs b/drivers/gpu/drm/tyr/driver.rs
index 1ed8293a02..88a6ca27e9 100644
--- a/drivers/gpu/drm/tyr/driver.rs
+++ b/drivers/gpu/drm/tyr/driver.rs
@@ -351,7 +351,7 @@ fn drop(self: Pin<&mut Self>) {
 }
 
 const INFO: drm::driver::DriverInfo = drm::driver::DriverInfo {
-    major: 0,
+    major: 1,
     minor: 0,
     patchlevel: 0,
     name: c_str!("panthor"),
-- 
2.51.0

